charts/gardener/operator/templates/deployment.yaml

```diff
@@ -140,10 +140,6 @@ spec:
         {{- end }}
         - name: gardener-operator-config
           mountPath: /etc/gardener-operator/config
-{{- if .Values.hostAliases }}
-      hostAliases:
-{{ toYaml .Values.hostAliases | indent 6 }}
-{{- end }}
```

Comment:
Why this was removed? The Helm chart is of "general purpose" (not only used for local dev), so probably we should keep allowing this configuration?

Reply:
This is the only Helm chart where we added `hostAliases` and we could not find any other use case for it.
Is there a concrete purpose you have in mind?

Reply:
No, but that doesn't mean that nobody else has one :) I was just wondering why this was removed since it doesn't "hurt" to have this configurable.

Reply:
I added it again